README.md

```diff
@@ -70,7 +71,49 @@ Before beginning a sprint:
 
 During a sprint we track progress using a [scrum or kanban-like board](#notes)<sup>4</sup> with status columns for To Do, In Progress, Blocked, Ready for Review, Ready for QA, and Done. 
 
-After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
+After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
+
+## Velocity Tracking
```

Comment:
Maybe we can connect this to the overarching narrative a bit more - 

Let's say your team has broken user stories into tasks that are each ticketed, pointed, and assigned to team members. Now it's time to decide, how much should your team commit to as a whole for the next sprint? How does that translate to the total number of points that each team member should be assigned for the next sprint? Velocity tracking can help you do that by using a team's average historical output to inform how much work (measured in total number of points) to commit to in the upcoming sprint.